docs/configuration.md

```diff
@@ -1470,6 +1470,14 @@ Apart from these, the following properties are also available, and may be useful
         </td>
     </tr>
     <tr>
+        <td><code>spark.ssl.keyStoreBase64</code></td>
+        <td>None</td>
+        <td>
+            A base64 encoded key-store file. If this property is defined, spark.ssl.keyStore
```

Comment:
Instead of this, we should throw an error if both are defined.  That should never happen.  Fail fast.